ai/STA/Action/ProtectGoal.py

```diff
@@ -69,7 +69,7 @@ def exec(self):
             destination_position = stayInsideCircle(destination_position, goal_position, self.maximum_distance)
 
         # Calcul de l'orientation de la pose de destination
-        destination_orientation = get_angle(destination_position, ball_position)
+        destination_orientation = (ball_position - destination_position).angle()
```

Comment:
bin je sais, weird thing is quand jali changer la premeire fois jlai mis du bon sens, mais ca chiait, quand j'ai inverser ca foncitonnait. maybe c dans l'implementation de .angle() que c inversé...